frontend/server/src/DAO/Submissions.php

```diff
@@ -103,6 +103,38 @@ final public static function countTotalSubmissionsOfProblemset(
         return \OmegaUp\MySQLConnection::getInstance()->GetOne($sql, $val);
     }
 
+    /**
+     * Gets the count of total runs sent by students (non-admins) to a given problemset
+     *
+     * @param list<array{user_id: int|null, role: 'admin'|'owner'|'site-admin', username: string}> $admins
+     */
+    final public static function countTotalStudentsSubmissionsOfProblemset(
+        int $problemsetId,
+        array $admins
+    ): int {
+        $adminsQuery = '(';
+        foreach ($admins as $admin) {
+            $adminsQuery .= "'{$admin['username']}',";
+        }
+        $adminsQuery = rtrim($adminsQuery, ',') . ')';
```

Comment:
aquí lo mejor es usar el mismo truco que en https://github.com/omegaup/omegaup/blob/12da725b5f21f508a8734eccaeef4f0978aba4fc/frontend/server/src/DAO/Problems.php#L378, porque si no, esto puede ocasionar SQL injections:

```suggestion
        $placeholders = join(',', array_fill(0, count($admins), '?'));
```

y así en L132 se puede concatenar ese arreglo con la lista de admins.